src/components/Logs.tsx

```diff
@@ -57,6 +57,10 @@ function Logs({
             }),
     ]);
 
+    const displayRestart =
```

Comment:
Would `restartDisplay` increase code clarity? There is nothing wrong with the original name. Sharing merely because my mind reversed the words after lingering for a moment.

Reply:
Nope. This is displaying a thing called display

Reply:
Ha okay. Not sure how I missed that detail but definitely good to know.